XenAdmin/MainWindow.cs

```diff
@@ -2713,7 +2714,15 @@ private void SetClientUpdateAlert()
         {
             updateAlert = Updates.ClientUpdateAlerts.FirstOrDefault();
             if (updateAlert != null)
+            {
                 relNotesToolStripMenuItem.Text = string.Format(Messages.MAINWINDOW_UPDATE_RELEASE, updateAlert.NewVersion.Version);
+                downloadLatestSourceToolStripMenuItem.Text = string.Format(Messages.DOWNLOAD_SOURCE, updateAlert.NewVersion.Version);

```

Comment:
I think these two are the other way round, it's downloadSourceToolStripMenuItem that's associated with the alert, right?

Also, downloadLatestSourceToolStripMenuItem is always there even if the check does not run automatically, so I believe we need to format it (in the construcotr alongside other items) using the current version.

Reply:
Good spot on the alert-associated tooltip. 

Now that I've swapped them though I don't understand what you mean about its always there. It will be there if there is updates.clientversions has something in it. It always should be the latest version of XC so if I have the latest version I should show the button in the help menu so I can download that source. It will be hidden if the xcupdates.xml fails to download or doesn't exist.